### PR TITLE
Native Angular Report Viewer docs

### PR DESCRIPTION
**Pull request by DimitarNikolovv06**

### CODE REVIEW

**Review by ighristov — APPROVED** [review]
Great job, Mitko! Thank you for your efforts!
Only minor typos found, left comments.

File: embedding-reports/display-reports-in-applications/web-application/native-angular-report-viewer/api-reference/options.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+	@Component({
+		selector: 'app-root',
+		templateUrl: './app.component.html',
+		styleUrls: ['./app.component.scss']
+	})
+	
+	export class AppComponent {
+		title = 'Native Angular Report Viewer Demo';
+	
+		@ViewChild('viewer') public viewer!: ReportingAngularViewerComponent;
```
[the remark this answers is not in the document]
Comment (DimitarNikolovv06) [reply embedding-reports/display-reports-in-applications/web-application/native-angular-report-viewer/api-reference/options.md:51]: The exclamation sign tells ts that the property will not be undefined. Without the sign, the type has to change to ReportingAngularViewerComponent | undefined

File: embedding-reports/display-reports-in-applications/web-application/native-angular-report-viewer/how-to-use-with-reporting-service.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+		providers: [],
+		bootstrap: [AppComponent]
+	})
+````
+
+
+1. Use the report viewer selector - `reporting-angular-viewer` in the component's template:
+
+	````HTML
+<reporting-angular-viewer
```
[the remark this answers is not in the document]
Comment (DimitarNikolovv06) [reply embedding-reports/display-reports-in-applications/web-application/native-angular-report-viewer/how-to-use-with-reporting-service.md:63]: No, this indentation in the beginning is required when listing steps, just MD things(or maybe it was docs-seed that had problem with this, I don't remember) :/

**Review by todorarabadzhiev — APPROVED** [review]
Looks good

**Pull request merged**